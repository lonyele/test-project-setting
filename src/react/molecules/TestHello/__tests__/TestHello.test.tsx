import { shallow } from 'enzyme';
import * as React from 'react';
import * as ReactDOM from 'react-dom';
import { create } from 'react-test-renderer';

import { TestHello } from '..';
import '../../../../setupTests';

describe('TestHello describe~~~', () => {
  it('should render without fail', () => {
    const div = document.createElement('div');
    ReactDOM.render(<TestHello message="Whatttt" />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('should render with corret message that has passed', () => {
    const wrapped = shallow(<TestHello message="ASdfsf!!!!!!!!!!!!!!" />);
    expect(wrapped.text()).toEqual('ASdfsf');
  });

  it('should snapshopt', () => {
    const wrapped = create(<TestHello message="snappp" />);
    expect(wrapped).toMatchSnapshot();
  });
});
