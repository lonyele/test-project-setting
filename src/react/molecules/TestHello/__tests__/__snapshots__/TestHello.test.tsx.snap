// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestHello describe~~~ should snapshopt 1`] = `
<h1
  className="sc-bdVaJa fGBmow"
>
  snappp
</h1>
`;
